tests/Unit/Css/CssDocumentTest.php

```diff
@@ -570,16 +570,20 @@ public function provideInvalidNonConditionalAtRuleWhichCausesException(): array
     public function provideInvalidNonConditionalAtRuleWhichDoesNotCauseException(): array
     {
         return [
-            '`@font-face` without `font-family`' => ['
-                @font-face {
-                  src: url("/foo-sans.woff2") format("woff2");
-                }
-            '],
-            '`@font-face` without `src`' => ['
-                @font-face {
-                  font-family: "Foo Sans";
-                }
-            '],
+            '`@font-face` without `font-family`' => [
+                '
+                    @font-face {
+                      src: url("/foo-sans.woff2") format("woff2");
+                    }
+            ',
```

Comment:
The closing quote should be indented like the opening quote
```suggestion
                ',
```